arthur/exts/systems/system_information.py

```diff
@@ -121,6 +138,51 @@ async def software(self, ctx: Context) -> None:
         program = lib9front.generate_buzzwords(bullshit)
         await ctx.reply(program)
 
+    @command(name="face")
+    async def face(
+        self, ctx: Context, resolution: int | None = 60, *, image_url: URLConverter | None = None
+    ) -> None:
+        """
+        Generate a system-compatible face file given.
+
+        If specified, resolution is the integer width and height to use for the generated image, defaulting to 100 (for a 100x100 image).
```

Comment:
The arg default is 60 for resolution, either this doc string needs changing to 60, or the arg needs changing to 100.